nautobot/dcim/models/racks.py

```diff
@@ -532,7 +533,7 @@ def get_power_utilization(self):
             or 0
         )
 
-        return int(allocated_draw_total / available_power_total * 100)
+        return (allocated_draw_total, available_power_total)
```

Comment:
Anywhere you return a tuple, please return a `collections.namedtuple` instead.

See: https://gist.github.com/lucasnad27/e93b8f316259be3299e5#clarify-multiple-return-values-with-named-tuples